circle.yml

```
@@ -366,6 +369,11 @@ workflows:
           name: "browsers node14.7.0-chrome84"
           dockerTag: "node14.7.0-chrome84"
           chromeVersion: "Google Chrome 84"
+      - build-browser-image:
+          name: "browsers node14.13.1-chrome84"
```

I think you forgot to fix the name here?
```suggestion
          name: "browsers node14.13.1-chrome86"
```

---

True, thanks for this.